Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -1071,4 +1071,65 @@ public function __clone()
             $this[$key] = clone $val;
         }
     }
+
+    /**
+     * Clear tokens in the given range
+     *
+     * @param int $indexStart
+     * @param int $indexEnd
+     */
+    public function clearRange($indexStart, $indexEnd)
+    {
+        for ($i = $indexStart; $i <= $indexEnd; $i++) {
```

Comment:
- preincrementation in general is one cycle faster, some precompilers done that for writer, eg ZendOptimizer, 5.4+, some not
- preincrementation use less memory
- following Symfony rule - keep consistency with whole project